### ISSUE TITLE
Extra space above TOC on scroll

### ISSUE THREAD

**Issue by ndiego** [WordPress/wporg-developer#314]
When scrolling down the page, there is some additional white space above the TOC. 

<img width="1777" alt="image" src="https://github.com/WordPress/wporg-developer/assets/4832319/e6f1344b-84ab-4093-990a-1170bc71be24">

**Comment by adamwoodnz** [comment]
This has become apparent because the ToC has moved down with the content to allow for the search bar. Would you expect it to scroll up a little before becoming sticky? Is this an MVP issue?

**Comment by adamwoodnz** [comment]
It looks like this will change when the search bar moves into the header ([design](https://www.figma.com/file/2WxlJFzMJvqPfZL1EkAOVp/Developer-Resources?type=design&node-id=3735-19404&mode=design&t=OuaweoW0B641WLTI-0)), as the ToC and content will move up to the top of `main` again.

**Comment by adamwoodnz** [comment]
Moving to on hold as this depends on #312 and decisions on sticky headers

**Comment by adamwoodnz** [comment]
I believe this will still be an issue after header changes land, but I think it is probably best addressed in https://github.com/WordPress/wporg-mu-plugins/issues/491 as well

**Comment by ndiego** [comment]
Yeah, I think we can address this one later once other elements land.

**Comment by renintw** [comment]
The conclusion here is to remove that additional space, right?

**Comment by renintw** [comment]
The conclusion here is to remove that additional space, am I correct?

**Comment by ndiego** [comment]
> The conclusion here is to remove that additional space, am I correct?

Yes. If possible, that would be great!

**Comment by adamwoodnz** [comment]
It really needs to be `position: sticky` rather than `fixed`, but the layout and JS doesn't support that so it's likely a pretty heavy refactor. I like @jasmussen's idea to make the chapter list and ToC independent scrolling columns as I think it would be much simpler to implement.

**Comment by adamwoodnz** [comment]
Actually seems not as complicated as I thought, should have something ready tomorrow.
